docs/src/main/asciidoc/datasource.adoc

```diff
@@ -511,154 +504,155 @@ public class TestResources {
 }
 ----
 
-This will allow you to test your application even when it's compiled into a native executable,
-while the database will run in the JVM as usual.
-
-Connect to it using:
+This will allow you to test your application even when it is compiled into a native executable while the database will run as a JVM process.
 
+* Connect to it using:
```

Comment:
Single-item list again.

Reply:
Yes, this is the single-item list this time. The convention is right, however, you are right that this should be a numbered step to be consistent with the rest of the doc. Will patch it.